relay-event-normalization/src/event.rs

```diff
@@ -1519,38 +1506,39 @@ mod tests {
         }
         "#;
 
-        let mut event = Annotated::<Event>::from_json(json).unwrap().0.unwrap();
+        let Annotated(Some(mut event), mut meta) = Annotated::<Event>::from_json(json).unwrap()
+        else {
+            panic!("Invalid transaction json");
+        };
 
-        normalize_platform_and_level(&mut event);
+        normalize_default_attributes(&mut event, &mut meta, &NormalizationConfig::default());
 
-        insta::assert_ron_snapshot!(SerializableAnnotated(&Annotated::new(event)), {}, @r###"
-        {
-          "level": "info",
-          "type": "transaction",
-          "platform": "other",
-        }
-        "###);
+        assert_eq!(event.level.value().unwrap().to_string(), "info");
```

Comment:
```suggestion
        assert_eq!(event.level.as_str(), Some("info"));
```

Looking at the `Annotated` signatures, this should work.

Reply:
Updated the last entry. It's not possible as the others are not `String`s but enums and don't implement `AsRef`. `to_string` goes through the `Display` implementation, where we have the string mapping defined.